### DISCUSSION TITLE
COMET using Herbie and GOES2Go

### DISCUSSION THREAD

**Discussion by bryanguarente** [blaylockbk/Herbie#130] [Show and tell]
Brian,
For the last year or so, The COMET Program has been taking advantage of the Herbie and GOES2Go packages to grab and plot data for our lessons.  Most recently, we used it to make a bunch of ensemble plots necessary for one of our in-person classes.  I was easily able to grab ECMWF, GFS/GEFS, and NBM data with your packages.  It was a huge boon to us here at COMET... in fact, it was such a boon, that we are starting to build all of our in-house visualization around your package.  

Recently, I have added code to my local copy of your code to get the archive (not reforecast) GEFS data and added NWM data pulls (but that is incomplete for a number of reasons).  I would like to send some pull requests in the future, I just haven't gotten to it with the holidays and my computer needing to go in for repairs.  

We aren't necessarily the best coders in the world at COMET, but we are more than happy to have your Herbie and GOES2Go packages in our back pocket to help us with all the data we often need to pull.  

Just a big thank you I guess is what this is.  

If you are interested in seeing some of the work we have done with this, feel free to comment back or DM me.  

Thanks,
Bryan Guarente (and the rest of the COMET science staff)

**Comment by blaylockbk** [comment]
Hi @bryanguarente,
Wow, this was unexpected. I'm glad to hear Herbie and GOES2go are being used by COMET. I admire all the training material COMET produces. COMET has helped me *many* times.

I originally built Herbie and GOES2go because I needed easy and fast access to the HRRR model and GOES satellite data I was working with routinely as a graduate student. I'm pleased to hear they are being used to help you in that way too. These are projects I mostly work on and maintain in my free time now, but I welcome any pull requests! I'm excited to see how you have extended Herbie and hope to expand its capabilities. I don't claim to be a great coder either, but try to make improvements with each commit.

Thanks again for sharing this news, and it's good to "meet" you, Bryan.
